nss-day-cohort-30/bangazon-api-maniacal-magpies
# Product controller bn

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

--The product controller as well as product tests are complete.  All tests should pass, given that you have at least one Customer, Product, ProductType, and PaymentType in your database.  Use one of the following SQL statements to add your first product for testing purposes:

-- if you have no products yet
INSERT INTO Product (ProductTypeId, CustomerId, Price, Title, [Description], Quantity)
    VALUES (1, 1, 11.50, 'Electronic Thingy', 'It''s a thing and it''s electronic', 5);

-- if you already have a product
UPDATE Product 
    SET ProductTypeId = 1,
    CustomerId = 1,
    Price = 11.50,
    Title = 'Electronic Thingy',
    [Description] = 'It''s a thing and it''s electronic', 
    Quantity = 5
    WHERE id = 1;

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ProductController-BN`
3. Use Test Explorer to run all TestProducts tests.
4. run program...
5. verify in Postman if you like.  URL is /products


# Checklist:
